orca-applications/src/main/groovy/com/netflix/spinnaker/orca/applications/tasks/DeleteApplicationTask.groovy

```diff
@@ -33,7 +34,8 @@ class DeleteApplicationTask extends AbstractFront50Task {
   KeelService keelService
 
   @Override
-  TaskResult performRequest(Application application) {
+  TaskResult performRequest(Application application, StageExecution stage) {
+    String user = stage.execution.trigger.user ?: "anonymous"
```

Comment:
This shouldn't be necessary (and it should be `AuthenticatedRequest.getSpinnakerUser()` anyhow).

Reply:
See comment below. I don't think that works outside the context of... well... an authenticated request.

Reply:
Followed up on our Slack, it does work and the `keelService` just needs to be wired up with the correct interceptor.

Reply:
P.S. I noticed we're missing this line when configuring `keelService`:
```java
setRequestInterceptor(spinnakerRequestInterceptor)
```

Could that be it?

Reply:
Yeah, I think that's it. Kork's `SpinnakerRequestInterceptor` adds the headers, and the task handler sets the user in the context object it grabs values from.

Reply:
I've updated the PR with a fix.